Tests/regressionTests.js

```diff
@@ -0,0 +1,301 @@
+var homepage = require('../Pages/homePage');
+var loginpage = require('../Pages/loginPage');
+var fashionCategory = require ("../Pages/fashionCategoryPage");
+var itemPage = require ("../Pages/itemPage");
+var registerPage = require ("../Pages/registerPage");
+var aboutUs = require("../Pages/aboutUs");
+var termsandconditions = require("../Pages/termsAndConditions");
+var privacyAndPolicy = require ("../Pages/privacyAndPolicy");
+browser.waitForAngularEnabled(false);
+browser.ignoreSynchronization = true;
+browser.manage().window().maximize();
+var EC = protractor.ExpectedConditions;
+
+describe ("002: Registration", function (){
+    beforeEach(() => {
+        homepage.open("https://auct-app2-frontend.herokuapp.com/")
+        .then(() => homepage.clickOnCreateAnAccont())
+    })
+    
+    var random1 = Math.round(Math.random()*100);
+    var random2 = Math.round(Math.random()*100);
+
+    it ("001: User is able to create account with valid credentials", function (){
+        var firstName="Tester";
+        var lastName="123";
+        var email = `gigedov${random1}${random2}@getnada.com`;
+        var password ="testing1";
+        registerPage.enterFirstName(firstName)
+        .then(() => registerPage.enterLastName(lastName))
```

Comment:
fix identation here fot .then() part:

registerPage.enterFirstName(firstName)
        .then(() => registerPage.enterLastName(lastName))

Reply:
I added indentations for .then() part